server/game/cards/04.3-FFH/LordsportFisherman.js

```diff
@@ -7,7 +7,7 @@ class LordsportFisherman extends DrawCard {
                 onCardEntersPlay: event => event.card === this && event.playingType === 'marshal'
             },
             handler: () => {
-                let bottomCard = this.controller.drawDeck.last();
+                let bottomCard = this.controller.drawDeck.slice(-1)[0];
```

Comment:
You'd think JS would have a `top` or `peek` method on arrays, since it has `push` and `pop`. 😫